packages/debugger/lib/session/index.js

```diff
@@ -450,6 +450,18 @@ export default class Session {
     return await this.dispatch(actions.startFullMode());
   }
 
+  /**
+   * HACK warning!  This function modifies the debugger state
+   * and should only be used in light mode!
+   */
```

Comment:
Can you add to this comment with usage example? To include stubs to show:
1. Starting in lightMode
2. Invoking this method
3. Switching to full mode

Just to save time in the future :)